### DISCUSSION TITLE
I create a table! So where is it?

### DISCUSSION THREAD

**Discussion by boylesg** [leokhoa/laragon#594] [Q&A]
I created a table using the script below.

So where do I find it in here:

![image](https://github.com/leokhoa/laragon/assets/13853614/95de0b97-f5b7-4617-bc7a-245ed86426e1)

`CREATE TABLE Trades (

    ID int NOT NULL AUTO_INCREMENT KEY,
    Name varchar(255),
    Description varchar(255)
);

insert into Trades values (0,'Appliance Repair','Repair services for household appliances.');
insert into Trades values (0,'Asphalt and Paving','Asphalt and pavement installation and repair.');
insert into Trades values (0,'Bricklaying','Bricklaying for structures.');
insert into Trades values (0,'Cabinetry','Cabinet making and installation.');
insert into Trades values (0,'Carpentry','General carpentry and woodworking.');
insert into Trades values (0,'Concreting','Concrete pouring and finishing.');
insert into Trades values (0,'Construction','General contractors, builders and construction companies.');
insert into Trades values (0,'Demolition','Demolition services for structure removal..');
insert into Trades values (0,'Doors and Windows','Installation and repair of doors and windows..');
insert into Trades values (0,'Drywall and Plastering','Drywall installation and plaster work.');
insert into Trades values (0,'Electrical','Wiring, electrical installations and repairs.');
insert into Trades values (0,'Elevator Installation and Repair','Elevator installation and maintenance.');
insert into Trades values (0,'Excavation and earth moving','Operation of excavators and dump trucks.');
insert into Trades values (0,'Fencing (domestic);','Construction of paling, picket and Colorbond fences.');
insert into Trades values (0,'Fencing (commercial);','Construction of commercial paling and cyclone wire fences.');
insert into Trades values (0,'Fencing (rural);','Construction of fences on rural properties.');
insert into Trades values (0,'Fireplace and Chimney','Installation and maintnence of fireplaces and chimneys.');
insert into Trades values (0,'Flooring','Installation and repair of various flooring types.');
insert into Trades values (0,'Framing','Structural framing for buildings.');
insert into Trades values (0,'Gardening and Mowing','Garden maintenance and lawn mowing.');
insert into Trades values (0,'Glass and Glazing','Glass installation and repair services.');
insert into Trades values (0,'Gutters and Downspouts','Gutter and downspout installation and maintenance.');
insert into Trades values (0,'Handyman','Installation of shelves, doors, small painting jobs, plasterboard repairs and furniture assembly etc.');
insert into Trades values (0,'Home Inspection','Home inspection services for buyers and sellers.');
insert into Trades values (0,'HVAC (Heating, Ventilation, and Air Conditioning);','Ventilation and Air Conditioning);, Heating and cooling systems installation and maintenance.');
insert into Trades values (0,'Insulation','Installation and maintenance of insulation.');
insert into Trades values (0,'Interior Design','Interior design and decor services.');
insert into Trades values (0,'Land Management','Weed spraying, planting, slashing and woody weed removal etc.');
insert into Trades values (0,'Landscaping','Creation of simple gardens, inlcuding ongoing maintnence.');
insert into Trades values (0,'Landscape construction','Creation of gardens with structures.');
insert into Trades values (0,'Locksmith','Locksmith services for security needs.');
insert into Trades values (0,'Masonry','Bricklaying, stonework and concrete work.');
insert into Trades values (0,'Painting','Interior and exterior painting services.');
insert into Trades values (0,'Pest Control','Pest control services for homes and businesses.');
insert into Trades values (0,'Pet Grooming','Pet fur trimming, bathing and nail clipping etc.');
insert into Trades values (0,'Plumbing','Plumbing services including installation and repairs.');
insert into Trades values (0,'Pool and Spa Maintenance','Pool and spa installation and maintenance.');
insert into Trades values (0,'Renovation and Remodeling','Home renovation and remodeling services.');
insert into Trades values (0,'Roofing','Roofing installations and repairs.');
insert into Trades values (0,'Scaffolding','Scaffolding rental and setup services.');
insert into Trades values (0,'Security Systems','Installation and maintenance of security systems.');
insert into Trades values (0,'Septic Systems','Installation and maintenance of septic systems.');
insert into Trades values (0,'Siding','Siding installation and repair services.');
insert into Trades values (0,'Solar Panel Installation','Solar panel installation and maintenance.');
insert into Trades values (0,'Surveying','Land surveyors and mapping services.');
insert into Trades values (0,'Tiling','Ceramic, porcelain and other tile installations.');
insert into Trades values (0,'Welding','Welding for metalwork and repairs.');
insert into Trades values (0,'Window Installation and Repair','Window installation and repair services.');
insert into Trades values (0,'Window Cleaning','Domestic and commercial window cleaning.');`

**Comment by Pen-y-Fan** [comment]
Probably in the tab Query 2, the database will need to be created first. https://www.heidisql.com/forum.php?t=12945

**Comment by Kane0023** [comment]
After executing your SQL script to create the Trades table and insert data, you can find the table in your database management system (DBMS) interface, such as MySQL Workbench, phpMyAdmin, or any SQL command-line tool. To view the contents of the Trades table, run the query SELECT * FROM Trades; in your SQL interface, which will display all the records you just inserted, ensuring a [Total Quality](https://totalqualityexpress.com/) approach to data management. Make sure you are connected to the correct database where the Trades table was created.

**Comment by ppfoong** [comment]
Inside your system database `mysql`.

How do I know?

Both the schema databases have 0 byte, so they can be excluded as you have inserted some data into your table.

`sys` has 1 table and 100 views, which is the same as other MySQL setup. There is no structural change here.

So, the answer is `mysql`.

By right, you should create new database, and put your tables inside your own database, instead of making use of the system databases.

How to create database? Very simple.

Right click anywhere in Laragon window, go to MySQL, and click on Create database.

**Comment by BruTru** [comment]
> I created a table using the script below.

No, you tried to create a table. 
But the table needs to be into a database.
Where is the database?

And why do you try to insert id equal to 0 into an autoincrement field for which all values must be different?

First of all, i think you must learn basis of relational databases and MySQL.
